controller/controller.go

```diff
@@ -149,6 +151,10 @@ func (c *Controller) Boot() {
 	})
 }
 
+func (c *Controller) Booted() chan struct{} {
+	return c.booted
+}
```

Comment:
One issue we have is that in our integration tests we boot the controller and expect everything to work right away. It happens so far that we create a CR in K8s and expect an event which we want to count and this just breaks because of the asynchronous nature of the current implementation. Having `Booted` addresses this so you can just wait for everything set up properly while keeping the boot specific asynchronous actions running. I decided for a channel interface here to have a native workflow in case you just want to cancel it. Let me know what you think.